cicd/pipelines/configuring-security-context-for-pods.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: ASSEMBLY
+[id="configuring-security-context-for-pods"]
+= Configuring the security context for pods
+include::_attributes/common-attributes.adoc[]
+:context: configuring-security-context-for-pods
+
+toc::[]
+
+The default service account for pods that {pipeline-shortnames} starts is `pipeline`. The security context constraint (SCC) associated with the `pipeline` service account is `pipelines-scc`. The `pipelines-scc` SCC is similar to the `anyuid` SCC, but with minor differences as defined in the following YAML specification:
```

Comment:
```suggestion
The default service account for pods that {pipeline-shortnames} starts is `pipeline`. The security context constraint (SCC) associated with the `pipeline` service account is `pipelines-scc` as defined in the following YAML specification:
```
I think "The `pipelines-scc` SCC is similar to the `anyuid` SCC" was a little confusing. I think you can remove it completely.

Reply:
Unfortunately this is not true. The YAML specification specifices only the difference from the `anyuid` SCC. I have rephrased for clarity. (This text itself is there since the previous version)